templates/visuals/rvisual/pbiviz.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Why we need an empty file?

Reply:
If pbiviz wasn't specified, tools generates pbiviz with externalJS files.

Reply:
but pbiviz.json should exist as it declares metadata such as author name and email.

Reply:
tools gets externalJS files from this files, other metadata comes from the common template for pbiviz.json